plugins/qiskit_executor/schemas.py

```diff
@@ -115,39 +109,25 @@ class InputParametersSchema(FrontendFormBaseSchema):
         },
         validate=ma.validate.Range(min=1, min_inclusive=True),
     )
-    backend = EnumField(
-        QiskitBackends,
-        required=True,
-        allow_none=False,
-        metadata={
-            "label": "Backend",
-            "description": "The quantum computer or simulator that will be used.",
-            "input_type": "select",
-        },
-    )
     ibmqToken = ma.fields.String(
-        required=False,
+        required=True,
```

Comment:
The authentication parameters must be optional in the first step that starts the plugin execution.
See <https://qhana-plugin-runner.readthedocs.io/en/latest/plugin-types/circuit-executor.html#inputs> (`authentication details`).
Yes, this means that there are potentially two substeps, one for again asking for credentials and another to ask for the backend (if not already provided).

The authentication token (and the backend) can also already be part of the execution options. The execution options exist to make it easier to automate the circuit execution (as the circuits are often executed as part of an algorithm).

Reply:
The authentication is now optional in the first step. 
Currently, there is a maximum of 2 steps. If the user doesn't authenticate or doesn't select a backend, a second step is required. Should I implement 3 possible steps as you described?
> Yes, this means that there are potentially two substeps, one for again asking for credentials and another to ask for the backend (if not already provided).

Loading a token from the execution options isn't implemented yet. I'll add it in this PR.

Reply:
Yes, it should be three steps ([] marks optional inputs):
1. Circuit + Options + [Credentials + Backend]
2. Credentials + [Backend] (skipped if credentials are available from first step)
3. Backend (skipped if backend is available from first step)

Everything that can be specified in the first step (apart from the circuit) must also be possible to specify with the execution options only! This also includes the backend.

You can completely ignore username + password authentication. If there is a possibility to only use a user revokable API token, we will only use that token in the plugins (handling passwords is easier to mess up).